gradle.properties

```diff
@@ -1,3 +1,3 @@
 group=org.ballerinalang.mongodb
-version=4.2.0
+version=4.3.0
```

Comment:
Can't we release this as 4.2.1 (patch version) as we are only updating the graalVM compatibility status in here? WDYT @LakshanSS @AzeemMuzammil?

Reply:
Yes. We can go with `4.2.1`